access-front-door/README.md

```diff
@@ -1,3 +1,70 @@
 # Access Front Door
 
-This is the code required for opening the front door latch.
\ No newline at end of file
+This is the code required for opening the front door latch.
+
+
+## Setting up environment variables
+
+1. Copy or rename `access-front-door/src/env.example.py` to `access-front-door/src/env.py`
```

Comment:
❓ Do we need a `.gitignore` file added to stop folks committing `src/env.py`?

Reply:
I believe `access-front-door/src/env.py` has been added to `.gitignore`

Reply:
Aha, missed that this PR adds a root one 👍